templates/webapi/test/overview.html.ep

```diff
@@ -231,10 +231,17 @@
                         <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
                     </div>
                     <div id="add-comments-controls">
-                        <button type="submit" class="btn btn-danger">
-                            <i class="fa fa-comment"></i> Submit comment on all <%= scalar @$job_ids %> jobs
-                        </button>
-                        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Discard</button>
+		      <div class="form-check my-1">
+		      <input class="form-check-input"
+			     type="checkbox" value="" id="wannaRestartCheck">
+		      <label class="form-check-label" for="wannaRestartCheck">
+			Restart on Submit
```

Comment:
```suggestion
			Also restart job(s) on submit
```

Reply:
Can I avoid `also`? `Restart job(s) on submit` sound sufficient, no?

Reply:
I see the "also" as necessary to make it clear that the checkbox wouldn't "restart instead of commenting" but "restart *and* comment". But after the discussion we just had I suggest to actually have two buttons:
1. Left, less-alerting button for "comment", e.g. in light orange
2. Right, more-alerting button for "Comment and restart", e.g. the current red color